### PR TITLE
doc: update dev standup day to Monday

### PR DESCRIPTION
**Pull request by Rakshith-R**
This commit updates the standup day of the
week to Monday as agreed upon.

**Checklist:**

* [ ] **Commit Message Formatting**: Commit titles and messages follow
  guidelines in the [developer
  guide](https://github.com/ceph/ceph-csi/blob/devel/docs/development-guide.md#commit-messages).
* [ ] Reviewed the developer guide on [Submitting a Pull
  Request](https://github.com/ceph/ceph-csi/blob/devel/docs/development-guide.md#development-workflow)
* [ ] [Pending release
  notes](https://github.com/ceph/ceph-csi/blob/devel/PendingReleaseNotes.md)
  updated with breaking and/or notable changes for the next major release.
* [ ] Documentation has been updated, if necessary.
* [ ] Unit tests have been added, if necessary.
* [ ] Integration tests have been added, if necessary.

---

<details>
<summary>Show available bot commands</summary>

These commands are normally not required, but in case of issues, leave any of
the following bot commands in an otherwise empty comment in this PR:

* `/retest ci/centos/<job-name>`: retest the `<job-name>` after unrelated
  failure (please report the failure too!)

</details>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -186,7 +186,7 @@ More details are available [here](https://github.com/ceph/ceph-csi/issues/463)
 
 ## Dev standup
 
-A regular dev standup takes place every [Tuesday at
+A regular dev standup takes place every [Monday at
 12:00 PM UTC](https://meet.google.com/vit-qdhw-nyh) Convert to your local
```
Comment (Madhu-1) [inline README.md:190]: Can you check the time and confirm again?

Comment (Rakshith-R) [reply README.md:190]: Yes, of course I did when i posted the pr.

**Review by Madhu-1 — APPROVED** [review]

**Review by nixpanic — APPROVED** [review]
@Rakshith-R, can you check with @Thingee how to update the Ceph Community calendar too?

**Comment by Rakshith-R** [comment]
@mergifyio queue

**Pull request merged**